### PR TITLE
Added Django REST Swagger

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-6808

Review #151 first

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review

File: credentials/apps/api/v2/views.py [hunk 1]
```diff
@@ -19,15 +19,53 @@ class CredentialViewSet(viewsets.ModelViewSet):
...
+        ---
+        omit_parameters:
+            - query
+        """
+        return super(CredentialViewSet, self).retrieve(request, *args, **kwargs)
+
+    def update(self, request, *args, **kwargs):
+        """ Update a credential.
+        ---
+        omit_parameters:
```
Comment (rlucioni) [inline credentials/apps/api/v2/views.py]: What does `omit_parameters` do?

Comment (clintonb) [inline credentials/apps/api/v2/views.py]: This prevents the query parameters that would show up due to the `filter_class`. We only need form parameters (from the serializers).

Comment (rlucioni) [inline credentials/apps/api/v2/views.py]: Maybe I'm misunderstanding, but wouldn't it be good to have query parameters from the `filter_class` (`UserCredentialFilter`, in this case) appear in the docs? Examples being `program_uuid`, `status`, and `username`.

Comment (clintonb) [inline credentials/apps/api/v2/views.py]: For the list endpoint, yes. However, those filters don't apply when modifying a single instance (obtained via UUID).

Comment (rlucioni) [inline credentials/apps/api/v2/views.py]: Oh, I completely missed that these were endpoints for modifying single instances. Thanks for the clarification!

**Pull request merged**